React/my-app/src/components/Description_Box/styles.css

```diff
@@ -0,0 +1,31 @@
+.description_box {
+  display: flex;
+  flex-flow: column wrap;
+}
+
+.book_title {
+  font-size: 20px;
+  font-weight: bold;
+  margin-bottom: 20px;
+}
+
+.other_info {
+  font-size: 16px;
+  margin-bottom: 20px;
+}
+
+.description {
+  line-height: 19px;
+  color: #999999;
+  margin-bottom: 41px;
+  max-width: 563px;
+}
+
+.rent_button {
+  background: #00adee;
+  color: #fff;
```

Comment:
#FFF

Reply:
The prettier auto corrects me the #FFF and rewrites it as #fff

Reply:
okas, but keep in mind that we write on MAYUS by convention